charts/orchestrator/values-rc.yaml

```diff
@@ -0,0 +1,22 @@
+isReleaseCandidate: true # Indicates RC builds should be used by the chart to install RHDH and Sonataflow
```

Comment:
Same comment as in the `values.yaml` file: I think this field should be added to the RHDH and SonataFlow sections because it doesn't have a global scope.